project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.13
\ No newline at end of file
+sbt.version=1.4.7

```

Comment:
The sbt version is on 1.3.x deliberately for now because a change in the error reporting breaks our internal TeamCity integration

Reply:
Ok I reverted this change